lib-injection/sitecustomize.py

```diff
@@ -185,6 +202,21 @@ def _inject():
         _log("ddtrace_pkgs path is %r" % pkgs_path, level="debug")
         _log("ddtrace_pkgs contents: %r" % os.listdir(pkgs_path), level="debug")
 
+        if not executable_is_compatible():
+            _log("Found incompatible executable: %s." % sys.argv, level="debug")
```

Comment:
We might not want to do `sys.argv` since there might be sensitive data in the arguments.

Can we capture whether it was argv[0] or argv[1] which was the offender?